File: backend/SpringBoot/src/main/java/com/myApiLM/SpringBoot/Model/Educacion.java
package com.myApiLM.SpringBoot.Model;

import javax.persistence.Basic;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import lombok.Getter;
import lombok.Setter;


@Getter  @Setter
@Entity
public class Educacion {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    Long id;
    @Basic
    String school;
    String title;
    String img;
    String career;
    Integer start;
    Integer end;
    Long idPersona;

    public Educacion(Long id, String school, String title, String img, String career, Integer start, Integer end, Long idPersona) {
        this.id = id;
        this.school = school;
        this.title = title;
        this.img = img;
        this.career = career;
        this.start = start;
        this.end = end;
        this.idPersona = idPersona;
    }

    public Educacion() {
    }

    
}

File: frontend/src/app/entidades/educacion.ts
export class Educacion {
  

    id:number;
    title: string;
    school: string;
    start: number;
    end: number;
    img:string;
    career: string;
    idPersona:number;

    constructor (id:number,title:string,school:string, img:string,start:number,end:number, career:string , idPersona:number )
    {
        this.id=id;
        this.title=title;   
        this.school=school;
        this.img=img;
        this.start=start;
        this.end=end;
        this.career=career;
        this.idPersona=idPersona;
    }

}